src/DGtal/dec/doc/packageDEC.dox

```diff
@@ -60,14 +62,58 @@ Basic operators, such as Hodge duality operator or exterior derivative, can be c
 @b Related @b documentation @b pages
 
 @b Related @b examples
-- testDiscreteExteriorCalculus.cpp
+- exampleDECSurface.cpp
+- examplePropagation.cpp
 - exampleDiscreteExteriorCalculusUsage.cpp
 - exampleDiscreteExteriorCalculusSolve.cpp
-- examplePropagation.cpp
+- exampleDiscreteExteriorCalculusChladni.cpp
+- testDiscreteExteriorCalculus.cpp
 - testLinearStructure.cpp
 - testEmbedding.cpp
-- exampleDECSurface.cpp
 
 */
 
+
+
+/**
+   Example of Discrete Exterior Calculus with Chladni.
```

Comment:
> @kerautret, we've discussed with @rolanddenis about doxygen issues in the documentation. To clarify the "@example" file description, what do you think to:
ask the authors to document all example files

yes sure, now we can easily see with this PR which example should be completed.
But not sure that it implies a lot of doc comments since in general they are well given in the main module doc. Perhaps just to check if the short description of the example is ok and to check if the link to the doc module part exist.


> move the doxygen example file description from the module pages into the example file (similarly to what you did in the tools)

yes, It should be a good idea and it could be more easy to check if all examples where documented ;)